Scenario Based Samples/Save-and-SaveAs-Filter-View/README.md

```diff
@@ -0,0 +1,225 @@
+
+      ![save-new-view-icon-click](https://github.com/boldbi/samples/assets/92368448/acb09629-b6b9-4675-a879-91c7a6cd07be)
+
+      ![save-view-dialog](https://github.com/boldbi/samples/assets/92368448/9e5142f9-1ad9-4d7b-bb80-a4312fca7126)
+
+  3. Retrieve the name of the view from the `Save View` dialog and save it using the [saveFilterView]() method available in BoldBI.
+
+      ![save-view-dialog-viewname](https://github.com/boldbi/samples/assets/92368448/c570f75d-a504-44f7-b437-f9748a62a583)
+
+      ![save-view-filter-method](https://github.com/boldbi/samples/assets/92368448/6b07b000-89f1-4953-9352-f21a16193239)
+
+  4. The `Filter Overview` panel will get updated with the saved view.
+
+      ![updated-filter-overview-savedview](https://github.com/boldbi/samples/assets/92368448/b6bea3b0-eea3-4184-9ce0-98dcce0227d5)
+
+>**NOTE**: Please refer the [REST API](https://help.boldbi.com/server-api-reference/v4.0/api-reference/#operation/ItemViews_AddItemView) for know more details to save the filter view in the dashboard.
+
+### Save as Filter Views
+  1. To open the `Save View` dialog, use the [onSaveAsFilter]() event in BoldBI create method as follows:
+
+      ![onsaveasfilter-event](https://github.com/boldbi/samples/assets/92368448/20794174-a954-491a-baf7-d9a1fea9d762)
+
+  2. When the `Save As` icon is clicked in the filter overview, the `onSaveAsFilter` event will be triggered, and a dialog box will be created using ej2 controls.
+
+      ![save-as-icon-click](https://github.com/boldbi/samples/assets/92368448/87deb53f-ff4f-4319-a065-42ad453f562c)
+
+  3. Retrieve the name of the view from the `Save View` dialog and save it using the [saveAsFilterView]() method available in BoldBI.
+
+      ![save-as-view-dialog-viewname](https://github.com/boldbi/samples/assets/92368448/ec907f6b-3289-4aa7-95f9-d161c3904464)
+
+      ![saveas-filter-view-method](https://github.com/boldbi/samples/assets/92368448/ea6123be-8425-4f03-858a-18b92347eac2)
+
+  4. The `Filter Overview` panel will get updated with the saved view.
+
+      ![updated-filter-overview-savedasview](https://github.com/boldbi/samples/assets/92368448/7ab2009a-6bd9-40bc-9814-30ab4a9906bb)
+
+>**NOTE**: The same method will be used to save a new filter view and save as the filter view from an existing one.
+
+### Update Filter Views
+  1. To update a saved filter view in the dashboard, `modify` the saved filter views by applying filters in the view.
+
+      ![modify-filter-view](https://github.com/boldbi/samples/assets/92368448/4b43cc57-201b-426a-b9eb-b0d145bd2ac4)
+
+  2. By clicking the `Save` icon in the filter overview, the filter view will be updated using the [updateFilterView]() method available in BoldBI.
+
+      ![save-exist-view-icon-click](https://github.com/boldbi/samples/assets/92368448/89dc5363-d3ca-4987-acef-473a817d703c)
```

Comment:
There is no change in updating the filterviews in image. Please update the values and show in the second image.

Reply:
Added the image for updating the filter values in the dashboard and resolved the concerns